src/main/java/com/blackducksoftware/integration/fortify/batch/util/MappingParser.java

```diff
@@ -62,8 +70,16 @@
 
     private final static String Q_connector = "+and+";
 
+    private final static String DYNAMIC_SCAN_REQUEST = "DYNAMIC_SCAN_REQUEST";
+
     private final static Logger logger = Logger.getLogger(MappingParser.class);
 
+    private final SpringConfiguration springConfiguration;
```

Comment:
I have removed the Spring Configuration file and moved all the bean definition to the BlackDuckFortifyJobConfig where I have passed the required dependencies to the appropriate constructors. Can you please review it.